functions/wp-functions.php

```diff
@@ -251,3 +251,42 @@ function filter_oembed_attributes($return, $data, $url) {
     return str_replace('<iframe', '<iframe title="'. $title . '"', $return);
 }
 add_filter( 'oembed_dataparse', 'filter_oembed_attributes', 10, 4 );
+
+/*
+ * Update home url behavior for Flywheel. This is because Flywheel manually overides home url.
+ */
+if ( defined('FLYWHEEL_CONFIG_DIR') ) {
+    /*
+     * Update flywheel_home_url when Site Address(home) is updated
+     */    
+    function fuxt_update_home_url( $old_value, $new_value, $option ) {
+        update_option( 'fuxt_home_url ', $new_value, true );
+    }
+    add_action('update_option_home', 'fuxt_update_home_url', 10, 3);
+
+    /*
+     * Replace Site Address(home) with flywheel_home_url
+     */
+    function fuxt_get_home_url( $url ) {
+        $flywheel_home_url = get_option( 'fuxt_home_url ' );
+        return empty( $flywheel_home_url ) ? $url : $flywheel_home_url;
+    }
+    add_filter('option_home', 'fuxt_get_home_url');
+
+    /*
+     * Add inline style to enable home url edit and to change the description
+     */
+    function fuxt_enqueue_scripts($hook) {
+        if ( 'options-general.php' == $hook ) {
+            wp_add_inline_script(
+                'options_general',
+                `jQuery(window).load(function(){
+                    jQuery( "input#home" ).attr("readonly", false);
+                    jQuery( "input#home" ).parent().children("p").text("Enter the primary front end URL");
+                });`
```

Comment:
You can add this in the `admin.js` file here: https://github.com/funkhaus/fuxt-backend/blob/master/js/admin.js

The syntax for this is: https://github.com/funkhaus/fuxt-backend/blob/adc99b82a598c71106f816d0e4e8b29c244cf320/js/admin.js#L3-L5

This keeps the JS code out of PHP.

Reply:
We need that code run for the flywheel hosting only. If we include the code to admin.js, it will run for all other hosting too. That's why I used inline script

Reply:
OK, I think it would be good to move the code to admin.js. The code to remove readonly is extra for other hostings, but description replacement should be for all the hostings.

Reply:
Yeah we could inject a var `isFlywheel` into JS using the `wp_add_inline_script` and then do a test for it in `admin.js`? But it's probably overkill for now.